src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>7.3</LangVersion>
+    <RootNamespace>Microsoft.ML.AutoML</RootNamespace>
+    <IncludeInPackage>Microsoft.ML.AutoML</IncludeInPackage>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <MlDotNetPackageVersion>1.0.0</MlDotNetPackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="$(MlDotNetPackageVersion)" />
+    <PackageReference Include="Microsoft.ML.LightGBM" Version="$(MlDotNetPackageVersion)" />
+    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="$(MlDotNetPackageVersion)" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <!-- here we place this assembly into CLI package dir so it gets included in CLI nuget -->
```

Comment:
I think you can delete this commented out piece. We will need to rethink how we do the CLI packaging when it comes in. If we really need this again, we can get it from source control history.

Reply:
done